manifests/openshift-kube-conformance.yaml

```
@@ -0,0 +1,41 @@
+podSpec:
+  restartPolicy: Never
+  serviceAccountName: sonobuoy-serviceaccount
+  tolerations:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
+    operator: Exists
+  - key: CriticalAddonsOnly
+    operator: Exists
+  containers:
+    - name: report-progress
+      image: quay.io/mrbraga/openshift-tests-provider-cert:latest
```

@bostrt what about that change on all manifests?
```suggestion
      image: quay.io/mrbraga/openshift-tests-provider-cert:stable
```

---

I just create this `stable` tag pointing to `20220517094926`